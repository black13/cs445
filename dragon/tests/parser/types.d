program empty();
var a: string;
var b: char;
var c: boolean;
begin end.
