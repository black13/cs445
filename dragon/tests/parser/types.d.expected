PROGRAM `empty` WITH ARGS:
AND VARIABLES:
  DECLARATION WITH TYPE:
    STRING
  OF NAMES:
    `a`
  DECLARATION WITH TYPE:
    CHAR
  OF NAMES:
    `b`
  DECLARATION WITH TYPE:
    BOOLEAN
  OF NAMES:
    `c`
AND SUBPROGRAMS:
DOES:
-- done dumping ast --
